pori_python/ipr/main.py

```diff
@@ -526,10 +542,9 @@ def ipr_report(
 
     # SAVE TO JSON FILE
     if output_json_path:
-        if always_write_output_json or not ipr_result:
```

Comment:
if always_write_output_json is false don't write the json even though there's a default path